Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -643,6 +657,11 @@ void HTMLModelElement::sizeMayHaveChanged()
 
 void HTMLModelElement::configureGraphicsLayer(GraphicsLayer& graphicsLayer, Color backgroundColor)
 {
+#if ENABLE(MODEL_ELEMENT_IMMERSIVE)
+    if (!m_modelRendersInline)
+        return graphicsLayer.removeModelContents();
```

Comment:
Rather than doing this directly, this would make more sense to do via the ModelPlayer, as each player may have a different notion of how the graphics layer is configured.

Could this bit, m_modelRendersInline, be added to the ModelPlayerGraphicsLayerConfiguration struct and passed to configureGraphicsLayer instead?